aries_cloudagent/did/tdw/tests/test_routes.py

```diff
@@ -0,0 +1,66 @@
+from unittest import IsolatedAsyncioTestCase
+
+from aries_cloudagent.admin.request_context import AdminRequestContext
+from aries_cloudagent.core.in_memory.profile import InMemoryProfile
+from aries_cloudagent.did.tdw.routes import resolve_did_tdw
+from aries_cloudagent.resolver.did_resolver import DIDResolver
+from aries_cloudagent.tests import mock
+
+mock_response = {
+    "@context": [
+        "https://www.w3.org/ns/did/v1",
+        "https://w3id.org/security/multikey/v1",
+    ],
+    "id": "did:tdw:Qma6mc1qZw3NqxwX6SB5GPQYzP4pGN2nXD15Jwi4bcDBKu:domain.example",  # noqa: E501
+}
+
+
+class TestAnoncredsRoutes(IsolatedAsyncioTestCase):
```

Comment:
TestAnonCredsRoutes I assume is just a typo?

Reply:
ugh. copy/paste error.